README.md

```diff
@@ -134,7 +134,7 @@ resources:
   - kms:
       name: sdkms-plugin
       endpoint: unix:///var/run/kms-plugin/socket
-      cachesize: 100
```

Comment:
Perhaps I'm missing context, but why are we removing the `cachesize` and instead specifying `apiVersion: v2`? 

Is the cache no longer necessary since we are polling the status?

Reply:
For KMS v2 cachesize is not supported anymore, as they cache all the key instead of havinig a limit on how many key to cache. Also, apiVersion need to be passed to differentiate between v1 and v2. For more detail see doc: https://kubernetes.io/docs/tasks/administer-cluster/kms-provider/#configuring-the-kms-provider-kms-v2

Reply:
Ah I see, ok that makes way more sense now. Thanks!